src/panels/lovelace/editor/view-editor/hui-view-background-editor.ts

```diff
@@ -144,6 +144,7 @@ export class HuiViewBackgroundEditor extends LitElement {
         .computeLabel=${this._computeLabelCallback}
         @value-changed=${this._valueChanged}
         .localizeValue=${this._localizeValueCallback}
+        style=${`--picture-opacity: ${background.opacity! / 100};`}
```

Comment:
Can it be undefined?

Reply:
No it's set to `33` or `100` always.

Reply:
Use `styleMap` for this.

Reply:
`${(background.opacity || 100) / 100}`

Reply:
> No it's set to `33` or `100` always.

It can be undefined, def on first render

Reply:
Can we have `(background.opacity ?? 100) / 100`? For now, the code assumes that's defined but it can be `undefined` according to the typing;